### ISSUE TITLE
Export SVG - data-cell-id

### ISSUE THREAD

**Issue by kpalatzky** [jgraph/drawio#4532]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Describe the bug**
When exporting diagrams as SVG from https://embed.diagrams.net/, a non-standard attribute `data-cell-id` is being added to all XML `g` elements. This attribute is not part of the SVG standard specification.

**To Reproduce**
1. Open [diagrams.net](https://embed.diagrams.net/)
2. Add a simple shape (e.g., a circle) to the diagram
3. Export the diagram as SVG
4. Inspect the exported SVG file

**Expected behavior**
The exported SVG should only contain standard SVG attributes and not include the non-standard `data-cell-id` attribute on `g` elements.

**Screenshots**
![Screenshot 2024-07-10 093350](https://github.com/jgraph/drawio/assets/18258842/e2af522e-1690-4adb-951e-d30c634a4b01)

**draw.io version (In the Help->About menu of the draw.io editor):**
- draw.io version v24.6.4

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge
 - Browser Version 126.0.2592.87

**I tested the problem in incognito/private mode with all browser extensions switched off, write "yes" below:**
- Yes

**Additional context**
This non-standard attribute may cause issues with SVG parsers or viewers that strictly adhere to the SVG specification. It could also potentially increase file size unnecessarily. Consider removing this attribute during the SVG export process or providing an option to exclude non-standard attributes.

Is there an XSD that is being tested against? This ensures that the output XSD is valid when testing the software.

**Comment by davidjgraph** [comment]
Does it cause specific problems, are there any SVG renderers that fail to render it?

**Comment by marnovdm** [comment]
We upgraded our local draw.io instance (based on https://github.com/jgraph/docker-drawio) from v24.7.17 to v26.0.2 and noticed the exported SVG's no longer have a data-cell-id property, which we relied on for our integration with the SVGs to render live status updates (it's for a monitoring application). I can't find mention of this in any changelogs, I only found this issue where it's closed as not planned. Is there any other way to get an ID into the SVG again with the current versions?

EDIT: I just found out the data-cell-id is included if I export it as SVG with the option 'include copy of diagram' manually from the draw.io iframe on my page. However when using a postMessage export with type 'xmlsvg' these data-cell-id's are no longer included, while they were included in v24.7.17. Any way to get an export with postMessage working again with included data-cell-id's?
